### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by spaquet**
Added 2 occupy / skeleton libs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -670,14 +670,16 @@ vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https:/
...
-*Loaders / spinners / progress bars — Let the user know that something is loading*
+*Loaders / spinners / progress bars / Skeleton — Let the user know that something is loading*
 
  - [vue-radial-progress](https://github.com/wyzant-dev/vue-radial-progress) - Radial progress bar component for Vue.js.
  - [vue-simple-spinner](https://github.com/dzwillia/vue-simple-spinner) - A simple, flexible spinner for Vue.js
  - [vuex-loading](https://github.com/f/vuex-loading) - Multiple Loader Management for Vue/Vuex applications.
  - [vue-spinner-component](https://sergeyloysha.github.io/vue-spinner-component/) - Customizable, lightweight spinner for Vue.js.
  - [vue-progress-path](https://github.com/Akryum/vue-progress-path) - Customizable progress indicators and spinners that support any custom SVG path.
  - [vue-blockui](https://github.com/realdah/vue-blockui) - BlockUI for vue 2, similiar to jquery blockUI, can be used for loading screen.
+ - [](https://jrainlau.github.io/vue-occupy/) - Minimalist vue 2 occupy / skeleton
```
Comment (kazupon) [inline README.md]: Please add repo name.
And I seems it' not correct. 
not `Minimalist vue 2 occupy / skeleton` but `A Vue directive for occupying content places before the data has been loaded. `.

Comment (spaquet) [reply README.md]: This is the landing page of the repo name on github.

Comment (spaquet) [reply README.md]: ok for the other change

**Comment by kazupon** [comment]
close due to in-activity.
if you hope push this PR again, you can push it as new PR. :)

**Pull request closed without merging**